flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorITCase.java

```diff
@@ -78,22 +81,14 @@ public void teardown() throws Exception {
 	}
 
 	/**
-	 * Tests that a job will be re-executed if a new TaskExecutor joins the cluster.
+	 * Tests that a job can be re-executed after the job has failed due
+	 * to a TaskExecutor termination.
 	 */
 	@Test
-	public void testNewTaskExecutorJoinsCluster() throws Exception {
+	public void testCleanClusterStateAfterTaskExecutorTermination() throws Exception {
```

Comment:
The name is not very intuitive, I think testJobReExecutionAfterTaskExecutorTermination would be better.

Reply:
Good idea. I will apply the changes.